src/Beacon.jsx

```diff
@@ -14,16 +14,17 @@ const TOOLTIP_TOLERANCE = 50;
 
 function deepCloneNode(node, parent, fn) {
   const clone = fn(node, parent);
-  const childNodes = [...node.childNodes];
+  const childNodes = Array.prototype.slice.call(node.childNodes);
```

Comment:
Can you please elaborate? Looks like a magic to me and definitely worth adding a comment here.